RetailSuiteAccounting/UserInterface/02a_DebtorsCreditors.md

```diff
@@ -0,0 +1,2394 @@
+
+- *SKU*  
+Stock Keeping Unit. Product identification code.
+
+- *Product name*  
+Name of the product.
+
+- *Quantity*  
+Amount of products.
+
+- *Price*  
+Net price of product.
+
+- *Total price*  
+Total price of product.
+
+- *Date*  
+Date in which the operation has taken place.
+
+- *Posting type*  
+Type of operation. The following operations are available:  
+
+  - *Manual posting*
+  - *Reservation*
+  - *Waiting for picking*
+  - *Sale*
+  - *Order*
+  - *Purchase*
+  - *Back to the supplier*
+  - *Stocktaking entry*
+  - *Stock transfer*
+  - *Return*
+  - *Drop shipment*
+  - *Problem*
+  - *Pre-reserve*
+
+[comment]: <> (List aus Lager/Buchungshistorie)
+
+- *Warehouse*  
+Warehouse where the product is stored.
+
+- *Storage shelf at the time of posting*  
+Storage shelf where the product is located at the time of the operation.
+
+- *Storage shelf*  
+Storage shelf where the product is usually located.
+
+- *Movement*  
+Stock movement.
+
+- *Total stock*  
+Total stock of the product.
+
+- *Total available*  
+Total stock available.
+
+- *Comment*  
+Operation-related comment entered in the system, if any. By default, the document number is displayed.
+
+[commnet]: <> (Wo kommt dieses Kommentar her? Lager? Warehouse? Add Link, wenn verfübar?)
+
+
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
+ Click this button to update the receipt list.
+
+- [RESET]  
+ Click this button to reset the receipt list view.
+
+
+- [NEW/RESET]  
+Click this button to reset the *Customer/supplier* tab and create a new customer/supplier. The *Create customer/supplier* tab is displayed.
+
+- [DELETE CUSTOMER 'number']/[DELETE SUPPLIER 'number']  
+Click this button to delete the selected customer/supplier.
+
+- [SAVE]  
+Click this button to save all changes made in the selected customer/supplier.
+
+
+### Statistics
+
+*Accounting > Settings > Tab DEBTORS/CREDITORS > Tab Customer/supplier list > Select customer > Sub-tab Statistics*
+
+![Statistics](../../Assets/Screenshots/RetailSuiteAccounting/Settings/CustomerSupplier/Statistics.png "[Statistics]")
+
+> [Info] This sub-tab is only displayed when selecting a customer in the *Customer/supplier* list and the customer has already placed at least one order.
+
+
+- *Statistic type*  
+Click the drop-down list to select the statistics to be displayed. The following options are available:
+
+  - **General turnover**  
+  - **Turnover per product**
+  - **Number of units sold per product**
+
+  > [Info] The fields displayed vary depending on the option selected. When selecting an option, the *Statistics extras* side bar opens automatically.
+
+
+- ![Open](../../Assets/Icons/Back02.png "[Open]") (Open)  
+Click this arrow to open the *Statistics extras* side bar if necessary.
+
+
+**General turnover**  
+
+![General turnover](../../Assets/Screenshots/RetailSuiteAccounting/Settings/CustomerSupplier/Statistics01.png "[General turnover]")
+
```

Comment:
The list displays all statistics selected in the *Statistic type* drop-down list. All fields are read-only.